pythonCode/create_MP2RAGE_json.py

```diff
@@ -0,0 +1,136 @@
+"""
+WARNING:
+This script was created in October 2018 when the specification
+for MP2RAGE files was not fully finalized. Please double check with
+latest version of the specs to make sure this is accurate.
+
+This script will create the JSON files required for an MP2RAGE file.
+To function it requires a BIDS data set with properly named image files:
+The script will through all the folders of a BIDS data set and will create
+the JSON files in every folder where it finds a file ending
+with '_MPRAGE.nii.gz'.
+
+Created by RG 2018-10-03
+"""
+
+import os
+import json
+from shutil import copyfile
+from collections import OrderedDict
+
+'''
+DEFINE CONTENT OF JSON FILES
+'''
+# sub-*_inv-1_MPRAGE.json
+data_inv_1 = OrderedDict()
```

Comment:
One option that might be a little cleaner -- since you know all of the key-value pairs at the start, you could include all of them in the definition. So here, for `data_inv_1`, you might do:
```
data_inv_1 = OrderedDict([
      ('InversionTime', '900'),
      ('FlipAngle', '5')
])
```

Totally stylistic, but might be nice from a human readability perspective !